modules/core/src/main/java/org/apache/ignite/spi/metric/jmx/ReadOnlyDynamicMBean.java

```diff
@@ -57,16 +58,18 @@ else if ("invoke".equals(actionName))
 
     /** {@inheritDoc} */
     @Override public AttributeList getAttributes(String[] attributes) {
-        AttributeList list = new AttributeList();
-
+        AttributeList attrList = new AttributeList();
+        List<Attribute> list = attrList.asList();
         try {
-            for (String attr : attributes) {
```

Comment:
Please let's keep the old variable name

Reply:
Fine. Old name wasn't very good :) - it really should have been "name" to distinguish between the attribute instance, the attribute value and the attribute name.